heed/src/env.rs

```diff
@@ -288,7 +301,7 @@ pub struct Env(Arc<EnvInner>);
 
 impl fmt::Debug for Env {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let EnvInner { env: _, dbi_open_mutex: _, path } = self.0.as_ref();
+        let EnvInner { env: _, dbi_open_mutex: _, path, .. } = self.0.as_ref();
         f.debug_struct("Env").field("path", &path.display()).finish_non_exhaustive()
     }
 }
```

Comment:
Shall I add to the debug the Handle?

Reply:
No, I don't see a reason to show it but could you please use the explicit syntax, here? i.e. `handle: _`.

Reply:
Yeah good catch may avoid issue later!